### DISCUSSION TITLE
An /uptime command

### DISCUSSION THREAD

**Discussion by Roan-V** [PurpurMC/Purpur#430] [__Completed__] [answered]
Not really much to be said other than the title.
An /uptime command that would report current uptime of the server: months, days, hours etc
Possibly with last boot date (might be interesting: history of server boots?)

**Comment by BillyGalbreath** [comment, marked as answer]
Implemented in build 1233

https://github.com/pl3xgaming/Purpur/commit/2b65c49fc706c7651791c5d37018bb5da493329d
